<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shelterOwnerStats</class>
 <widget class="QWidget" name="shelterOwnerStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Stats">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Statistics</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="totalPet">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total Pets</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="totalPets">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>71</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #33ccff;
	border-radius: 35px;
}</string>
         </property>
         <property name="text">
          <string>#</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="adoptedPets">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total Adopted</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="totalAdopted">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>71</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #33ccff;
	border-radius: 35px;
}</string>
         </property>
         <property name="text">
          <string>#</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="pendingRequests">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total Pending</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="totalPending">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>71</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #ff6666;
	border-radius: 35px;
}</string>
         </property>
         <property name="text">
          <string>#</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="rejections">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total Rejected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="totalRejected">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>71</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #ff6666;
	border-radius: 35px;
}</string>
         </property>
         <property name="text">
          <string>#</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="percentageAdoption">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Percentage Adopted</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="percentage">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>71</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #88ff88;
	border-radius: 35px;
}</string>
       </property>
       <property name="text">
        <string>#</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
